docs/usage/frameworks/litestar.rst

```diff
@@ -303,20 +303,21 @@ You can access the database session from the controller by using the `db_session
         plugins=[alchemy],
     )
 
-Sessions in Middleware
-^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The idea of this section was to show how to access session that was already attached into the session's scope.

This is just showing how to create a custom middleware.  Is there something else you were trying to highlight?